Composer/packages/client/src/components/Modal/Confirm.jsx

```diff
@@ -41,9 +53,12 @@ const ConfirmDialog = props => {
         styles: dialogModal,
       }}
     >
-      {onRenderContent(subTitle, BuiltInStyles[style])}
+      <div css={[confirmationContainer, styles.content]}>
+        {onRenderContent(subTitle, BuiltInStyles[style])}
```

Comment:
So what if the caller could control whether or not this is disabled?

I'm envisioning something like:

```tsx
{onRenderContent({ subTitle, styles: BuiltInStyles[style], setDisabled })}
```
and in the call site:
```tsx
const setting = {
  disabled: true,
  onRenderContent: (props) => <DeleteConfirm {...props} />
};

function DeleteConfirm(props) {
  return <checkbox onChange={(checked) => props.setDisabled(!checked)} />
}
```

This way the component give's its callers a way to manage this state on their own. For instance, what if we have another modal that requires the user to type in the name of the bot to delete? Or if there is a dropdown the user must choose before continuing? This affords that flexibility.